setup.py

```diff
@@ -19,7 +19,7 @@
         'Operating System :: OS Independent',
     ],
     install_requires=[
-        'datadiff==2.0.0',
+        'testfixtures==6.14.1',
```

Comment:
Shouldn't `scrapy` be added here?

Also `six`, because Scrapy does not currently depend on it (unless it's removed like I suggested in another comment)

Reply:
Not sure. I find it redundant since you're supposed to already have `scrapy` as a dependency in your own project if you are trying to use this middleware in your Scrapy project. On the other hand, it could be useful to require a specific minimum Scrapy version.